NFTEA-Frontend/src/components/CreateAccountFront.vue

```diff
@@ -119,10 +140,42 @@
             </v-card>
           </v-col>
         </v-row>
-      </v-app>
-    </div>
+<!--      </v-app>-->
+<!--    </div>-->
   </v-container>
+</div>
 </template>
+
+
+<style>
+
+  .form {
+    background-color: #D1BE9C;
+    height: 100%;
+  }
+
+  .buttons {
+    column-count: 2;
+    height: 50px;
+  }
+
+  #cancelBtn {
+    border-radius: 20px;
+    background-color: #7D4F50;
+    height: 100%;
+    color: #D1BE9C;
+  }
+
+  #createBtn {
+    border-radius: 20px;
+    height: 100%;
+    background-color: #7D4F50;
+    color: #D1BE9C;
+  }
```

Comment:
If you want to, you can add some margin, padding & alter the font-weight to something bolder to make it look prettier. I don't know for the values, but something like:

```suggestion
  #cancelBtn {
    border-radius: 20px;
    background-color: #7D4F50;
    height: 100%;
    color: #D1BE9C;
    padding: 0.5rem 0.6rem;
    margin-top: 1rem;
    font-weight: bold; /* Depending on the font-family, these also go for actual values e.g. 400, 500, 600, etc.*/
  }

  #createBtn {
    border-radius: 20px;
    height: 100%;
    background-color: #7D4F50;
    color: #D1BE9C;
    padding: 0.5rem 0.6rem;
    margin-top: 1rem;
    font-weight: bold;
  }
```